hveto/utils.py

```diff
@@ -19,12 +19,19 @@
 """General utilities for hveto
 """
 
-from math import ceil
+import glob
+import warnings
 
+from math import ceil
 from gwdatafind.utils import filename_metadata
 
+from gwpy.table import (Column, EventTable)
+
+from . import const
+
 __author__ = 'Duncan Macleod <duncan.macleod@ligo.org>'
 __credits__ = 'Alex Urban <alexander.urban@ligo.org>'
+__credits__ = 'Siddharth Soni <siddharth.soni@ligo.org>'
```

Comment:
So that the names don't get overwritten, how about we change this slightly to append strings?
```python
__credits__ = ('Alex Urban <Alexander.urban@ligo.org> '
               'Siddharth Soni <siddharth.soni@ligo.org>')
```

Reply:
@alurban I don't think you meant to have A capital in Alexander.urban and there is space between > and ' at the end of your email id.

Reply:
@siddharth101, the capital A is a typo (which I have fixed), but the space is on purpose. It means that if you were to print the `__credits__` variable, you would see this:

```python
>>> print(__credits__)
Alex Urban <alexander.urban@ligo.org> Siddharth Soni <siddharth.soni@ligo.org>
```